CorrelatorSharp/packages.config

```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Thanks for clarifying Ivan, I reproduced that failure after removing the package from my local user .nuget cache.  Have added the packages.config back with my latest commit